webpack/assets/javascripts/react_app/components/LoginPage/LoginPageHelpers.js

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment } from 'react';
+import isEmpty from 'lodash/isEmpty';
+import { translate as __ } from '../../common/I18n';
+
+// eslint-disable-next-line consistent-return
+export const formatAlert = alerts => {
+  if (!isEmpty(alerts)) {
+    const alertType = Object.keys(alerts)[0];
```

Comment:
properties order in objects is not guaranteed in js, `alerts` is a plain object or a map? maybe it would be better to use a property named `type` from the first place?

Reply:
The objects that we receive from the server here looks like `warning: "some-warning.."` or `error: "some-error.."`
I will consider doing this seperation of `type` and `message`

Reply:
I think it would be much better to change the object structure at the server